README.md

```diff
@@ -5,13 +5,14 @@
 [![Anaconda-Server Badge](https://anaconda.org/scipp/scipp/badges/installer/conda.svg)](https://conda.anaconda.org/scipp/label/dev)
 [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/scipp/scipp-jupyter-demo)](https://hub.docker.com/r/scipp/scipp-jupyter-demo)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/scipp/scipp/master?filepath=docs%2Fgetting-started%2Fquick-start.ipynb)
+[![Questions](https://img.shields.io/github/issues/scipp/scipp/question?color=%23d876e3&label=questions)](https://github.com/scipp/scipp/issues?utf8=%E2%9C%93&q=label%3Aquestion)
 
 # Scipp
 
 See https://scipp.readthedocs.io for project context and how to use and install the **scipp** Python module.
 
 See [here](https://github.com/scipp/scipp/issues?utf8=%E2%9C%93&q=label%3Aquestion) for a list of frequently asked questions on how to use scipp.
-This is also the place to ask new questions that have not been answered yet, by opening a new issue with the [![question label](docs/images/question.png)](https://github.com/scipp/scipp/issues?utf8=%E2%9C%93&q=label%3Aquestion) label attached to it.
```

Comment:
Is `question.png` still used anywhere or could it be removed too?

Reply:
it's still used in the docs pages, as I thought having the inline image didn't work so well in the Readme but it does work quite nicely in the docs pages